### PR TITLE
removing unnecessary language attribute

### PR DESCRIPTION
**Pull request by dlockhart**
Couple minor fixes for lit-analyzer strict mode.

### CODE REVIEW

File: components/demo/code-view.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ class CodeView extends LitElement {
 	render() {
 		return html`
 			<div class="d2l-code-view-src"><slot @slotchange="${this._handleSlotChange}"></slot></div>
-			<div language="${this.language}" class="d2l-code-view-code">${this._codeTemplate}</div>
+			<div class="d2l-code-view-code">${this._codeTemplate}</div>
```
Comment (dlockhart) [inline components/demo/code-view.js:30]: This wasn't doing anything.

File: components/demo/demo-snippet.js [hunk 2]
```diff
@@ -24,8 +24,9 @@ class DemoSnippet extends LitElement {
 	}
 
 	render() {
+		const dirAttr = this._dir === 'rtl' ? 'rtl' : 'ltr';
 		return html`
-			<div class="d2l-demo-snippet-demo" dir="${this._dir}">
+			<div class="d2l-demo-snippet-demo" dir="${dirAttr}">
```
Comment (dlockhart) [inline components/demo/demo-snippet.js:29]: It was complaining that `dir` can't be any arbitrary string -- so forcing it to be one of these values fixes things.

File: components/demo/code-view.js (hunk 1 above)
Comment (dbatiste) [reply components/demo/code-view.js:30]: Currently I think this is used by:
```
:host .d2l-code-view-code::before {
    content: attr(language);
}
```

The language shows here:
image

Maybe we can just change it to a data attribute?

Comment (dlockhart) [reply components/demo/code-view.js:30]: Ah cool, I was having trouble figuring out how it got applied. Will fix!

Comment (dlockhart) [reply components/demo/code-view.js]: K fixed!

**Pull request merged**